### PR TITLE
Add image Suru strip to IoT digital signage page

### PR DESCRIPTION
**Pull request by MeltySnow**
## Done

Added image Suru strip to IoT digital signage page.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/internet-of-things/digital-signage
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Ensure that the design matches the [specification](https://github.com/canonical-web-and-design/web-squad/issues/1460)

## Issue / Card

Fixes canonical-web-and-design/web-squad#1460

## Screenshots
image

### CODE REVIEW

**Comment by anasereijo** [comment]
- [ ] delete the image that is on the top -the one with the screens. (sorry this is my fault)

- [ ] `u-align--center` the other image and remove the size `?w=368`

- [ ] make the "Learn more about digital signage on Ubuntu" section light grey 

- [ ] make the "Digital signage players need the power of Ubuntu" section white

- [ ] on mobile could we have the gradient as background? Like what we've done for /embedded

File: templates/internet-of-things/digital-signage.html [hunk 1]
```diff
@@ -7,26 +7,26 @@
...
         </div>
-        <div class="col-5">
-          <ul class="p-list">
-            <li class="p-list__item is-ticked">Easy out-of-the-box standard kiosk solution on Ubuntu</li>
-            <li class="p-list__item is-ticked">Secure, reliable and remotely upgradable</li>
-            <li class="p-list__item is-ticked">Rich ecosystem of hardware and software solutions</li>
-            <li class="p-list__item is-ticked">Hundreds of thousands of smart displays already deployed on Ubuntu</li>
-          </ul>
+        <div class="col-6 u-vertically-center u-image-position u-hide--small">
+          <img src="https://assets.ubuntu.com/v1/2c3ad93c-digital-signage-2.png?w=368" class="u-image-position--right"/>
```
Comment (solazio) [inline templates/internet-of-things/digital-signage.html]: please use `{{ ASSET_SERVER_URL }}` instead of `https://assets.ubuntu.com/v1/`

**Comment by MeltySnow** [comment]
I've pushed a change that should address all these issues.

**Comment by anasereijo** [comment]
@MeltySnow looks good!

**Pull request merged**